test/suspenders/cleanup/organize_gemfile_test.rb

```diff
@@ -0,0 +1,21 @@
+require "test_helper"
+require_relative "../../../lib/suspenders/cleanup/organize_gemfile"
+
+module Suspenders
+  module Cleanup
+    class OrganizeGemfileTest < ActiveSupport::TestCase
+      test "organizes Gemfile by group" do
+        original = file_fixture("gemfile_messy").read
+        modified = file_fixture("gemfile_clean").read
+        File.open("tmp/Gemfile", "w+") { _1.write original }
+
+        Suspenders::Cleanup::OrganizeGemfile.perform("tmp/Gemfile")
+
+        gemfile = File.read("tmp/Gemfile")
+        assert_equal modified, gemfile
+      ensure
+        File.delete("tmp/Gemfile")
+      end
```

Comment:
I originally tried using `TempFile` for this, but found it challenging, since the file was closed when I tried to make assertions.